packages/portfolio-contract/src/plan-solve.ts

```diff
@@ -87,6 +87,21 @@ export interface SolvedEdgeFlow {
 /** Model shape for javascript-lp-solver */
 export type LpModel = IModel<string, string>;
 
+/**
+ * Link to Factory and Wallet contracts:
+ * https://github.com/agoric-labs/agoric-to-axelar-local/blob/cd6087fa44de3b019b2cdac6962bb49b6a2bc1ca/packages/axelar-local-dev-cosmos/src/__tests__/contracts/Factory.sol
```

Comment:
This is decent, but I see no description here or there of what the Factory and Wallet contracts _are_/what they _do_/etc. Please take an action item to provide documentation of that.